src/frontend/komponenter/Fagsak/Saksoversikt/Behandlinger.tsx

```diff
@@ -56,26 +62,15 @@ const Behandlinger: React.FC<IBehandlingshistorikkProps> = ({ minimalFagsak }) =
         ),
     ];
 
+    const henlagtBehandlingEksisterer = behandlinger.some((behandling: BehandlingTabellobjekt) =>
+        erBehandlingHenlagt(behandling.behandlingEllerTilbakekreving.resultat as BehandlingResultat)
```

Comment:
Denne vil nok ikkje plukke opp henlagte tilbakekrevingar - der er typen _Behandlingsresultatstype_

Sjå Behandlinger.visRad-funksjonen for korleis det er gjort der, trur det er lurt å bruke samme logikk.